pkg/controllers/management/clusterprovisioner/provisioner.go

```diff
@@ -10,7 +10,7 @@ import (
 	"strings"
 	"time"
 
-	v32 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	apimgmtv3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
```

Comment:
Only real changes in this file are the package rename, and getting it to skip any eksV2 ?

Reply:
Correct